crossdock/client/ctxpropagation/behavior.go

```diff
@@ -58,6 +59,10 @@ func Run(t crossdock.T) {
 	assert := crossdock.Assert(t)
 	fatals := crossdock.Fatals(t)
 
+	tracer, closer := jaeger.NewTracer("crossdock", jaeger.NewConstSampler(true), jaeger.NewNullReporter())
+	defer closer.Close()
+	opentracing.InitGlobalTracer(tracer)
```

Comment:
The logic is setup so that if a tracer is not defined it falls back to the global tracer.  We can inject any tracer we want into yarpc by adding it to the yarpc.Config when we create the dispatcher.  I'm using it for two purposes here, one for setting up the tests with initCtx, and the other for YARPC.  I can split them to be individualized it we want.